python/pip_install/requirements.bzl

```diff
@@ -53,7 +53,7 @@ def compile_pip_requirements(
         generate_hashes: whether to put hashes in the requirements_txt file.
         py_binary: the py_binary rule to be used.
         py_test: the py_test rule to be used.
-        requirements_in: file expressing desired dependencies.
+        requirements_in: file(s) expressing desired dependencies.
```

Comment:
A recent PR deprecated the `requirements_in` arg in favor of `src` to allow toml files as inputs, so you'll need to sync to head and update the code in light of that.

Does passing multiple toml files also work? If not, that's fine. Just curious.